### PR TITLE
ref(ui): Convert CspHelp to Typescript

### PR DESCRIPTION
**Pull request by priscilawebdev**
Convert CspHelp to Typescript. The component looks like:

image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  };
+
+  const getLink = () => {
+    const href = getLinkHref();
+
+    return (
+      <React.Fragment>
+        <ExternalLink href={href}>developer.mozilla.org</ExternalLink>
+        <ExternalLink href={href} className="external-icon">
+          <IconOpen size="xs" />
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx:43]: Could this icon go inside the other link? Seems redundant to have two links.

File: src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  };
+
+  return (
+    <div>
+      <h4>
+        <code>{key}</code>
+      </h4>
+      <blockquote dangerouslySetInnerHTML={getHelp()} />
+      <StyledP>
+        <span>{'— MDN ('}</span>
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx:56]: Should this be an mdash?

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx:56]: I am not sure, so I prefer to leave as it is for now...

File: src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx:43]: sure.

File: src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ const CSPHelp = ({data: {effective_directive: key}}: Props) => {
       </h4>
       <blockquote dangerouslySetInnerHTML={getHelp()} />
       <StyledP>
-        <span>{'— MDN ('}</span>
+        <span>{'_ MDN ('}</span>
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/cspHelp/index.tsx:54]:
```suggestion
        <span>{'\u2014 MDN ('}</span>
```

**Review by matejminar — APPROVED** [review]

**Pull request merged**